.github/workflows/docs-production-netlify-cli.yml

```
@@ -23,6 +21,7 @@ defaults:
 jobs:
   check-version:
     name: Check Docs Version and Branch
+    if: ${{ github.ref != 'refs/heads/prod-docs-latest' }} # Exclude the prod-docs-latest branch from triggering the workflow
```

Shouldn't it be that way?
```suggestion
    if: ${{ github.ref != 'refs/heads/prod-docs/latest' }} # Exclude the prod-docs-latest branch from triggering the workflow
```

---

Fixed